hash/c64_cass.xml

```diff
@@ -16305,6 +16523,42 @@ Some are based on the C64 Tape Archive at https://archive.org/download/c64tapes/
 		</part>
 	</software>
 
+	<software name="wsbi" cloneof="wsb">
+		<description>World Series Baseball (Imagine)</description>
+		<year>1985</year>
+		<publisher>Imagine</publisher>
+
+		<part name="cass1" interface="cbm_cass">
+			<dataarea name="cass" size="454978">
+				<rom name="World_Series_Baseball.tap" size="454978" crc="89ab0eb1" sha1="2c4a8b547f1b8dcfa52f2899df3c0efadb7c8929"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="thewormi">
+		<description>The Worm in Paradise</description>
+		<year>1985</year>
+		<publisher>Level 9 Computing</publisher>
+
+		<part name="cass1" interface="cbm_cass">
+			<dataarea name="cass" size="465493">
+				<rom name="Worm_in_Paradise,_The.tap" size="465493" crc="4f3cc7b9" sha1="a8e027a8d8ff350816312eaf8adceec9f2fc2f3f"/>
```

Comment:
Please don’t move articles to the end like this in tape labels.

Reply:
The tape labels shown here are identical to the ones used in the C64 Ultimate Tape Archive V2.0.  I think I was asked not to change them in an earlier PR.

Reply:
Yeah, the trouble is, these other projects tend to mess with the labels in all sorts of ways, while MAME is supposed to preserve stuff.  I guess we put it down as something to fix later when giving the software list an overhaul.

Reply:
Agreed.